README.md

```diff
@@ -614,6 +614,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [report-uri.io](https://report-uri.io/) — CSP and HPKP violation reporting
   * [ringcaptcha.com](https://ringcaptcha.com/) — Tools to use phone number as id, available for free
   * [seclookup.com](https://seclookup.com/) - Seclookup APIs can enrich domain threat indicators in SIEM, provide comprehensive information on domain names, improve threat detection & response. Get 50K lookups free [here](https://account.seclookup.com/).
+  * [Selefra](https://www.selefra.io/) - An open-source policy-as-code software that provides analytics for multi-cloud and SaaS.
```

Comment:
This does not appear to be a SaaS, I see there is a "Selefra Cloud" and it offers a free trial but I dont see any details of what the trial is, how long it runs or anything like that.  

Please provide a link that explains the free tier of the SaaS - the only part that would be applicable to this list.

Reply:
[Selefra](https://app.selefra.io) This is our SaaS website. Our free version offers 1 project, 1 connection, and 1 member for users to use. New users are given a 7-day trial of the team version. However, our free version also allows full access to all functionalities, with only limited quantities. We currently do not have a designated pricing page.

Reply:
All I see there is the image below, no ability to verify that these are the features of the free accounts. I need to see somewhere on the site pricing details that show what you say.

![Screenshot 2023-03-20 at 12 11 07](https://user-images.githubusercontent.com/82342/226323142-3dd21011-e559-4ada-b0cc-59d2dfeaa75e.png)

Reply:
We have added a pricing page https://www.selefra.io/pricing